downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -294,9 +308,14 @@ All the connectors listed below have been released under new major versions, due
 
 - Removed the previously deprecated `nats:Message` record. Consequently, corresponding APIs no longer accommodate this record. Users are advised to transition to utilizing subtypes of `nats:AnydataMessage` for continued functionality.
 
-#### `aws.redshift` package
+#### `googleapis.gmail` package
 
-- Introduced seamless connectivity to Amazon Redshift databases.
+- Enhanced connector APIs by incorporating resource function syntax, along with improved documentation and examples.
+
+#### `twilio` package
+
+- Enhanced integration with Twilio services, now fully supporting all features offered through the Twilio Basic API version 2010-04-01.
```

Comment:
should this be `Enhanced integration with Twilio services, now fully supporting all features offered through the Twilio Basic REST API version 2010-04-01`?